exercises/2b.js

```diff
@@ -11,5 +11,6 @@
  * @returns {Array} The array in MB
  */
 export function convertFilesizes(arr) {
-  // TODO
+  const newNum = num => num/1000
+  return arr.map(newNum)
```

Comment:
I love it!

```js
return arr.map(num => num / 1000)
```

is also good in case you don't want to "name" the arrow function